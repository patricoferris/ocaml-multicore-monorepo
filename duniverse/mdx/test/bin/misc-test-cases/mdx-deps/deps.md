Toto

```ocaml file=tikitaka.ml
```

```ocaml file=tuktuk.ml,skip
```

```cram file=burn.sh
```

```sh dir=ping/
```
