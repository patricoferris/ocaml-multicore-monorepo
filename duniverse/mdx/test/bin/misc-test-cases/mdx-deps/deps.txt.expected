((4:file11:tikitaka.ml)(4:file7:burn.sh)(3:dir5:ping/))